java/ingest/ingest-runner/src/main/java/sleeper/ingest/runner/task/IngestJobRunner.java

```diff
@@ -75,7 +75,7 @@ public class IngestJobRunner implements IngestJobHandler {
     private final String taskId;
     private final StateStoreProvider stateStoreProvider;
     private final IngestJobTracker tracker;
-    private final AmazonSQS sqsClient;
+    private final SqsClient sqsClient;
```

Comment:
It doesn't look like this field is used. The same goes for instanceProperties. It looks like both fields can be deleted.